### ISSUE TITLE
Whitelist: saltwood.top

### ISSUE THREAD

**Issue by SALTWOOD** [pengelana/blocklist#194]
### Issue(s):

Type `x` in between `[ ]` and make sure there isn't any space between brackets. Example; for Your Selected Issue(s), type like this - `[x]` 
You can select more than one category of issues if you need to!

- [x] Whitelist
- [ ] Blacklist
- [ ] App Broken
- [x] Website Broken
- [x] Request
- [ ] Other Issue

<hr>

### Domain(s):

If you are submitting this issue for whitelist/blacklist issue, send us the domain(s) for whitelisting/blacklisting here. 

- Domain(s):

```
saltwood.top
```

<hr>

### Detail:

- Your Issue Detail:

I’m the owner of the domain saltwood.top, and I recently noticed it’s been added to an ad block blacklist. I believe this happened because I ran a BT tracker on it in the past. At the time, I didn’t realize the potential issues this could cause.

As soon as I became aware of the problem, I removed the BT tracker completely from my domain. I’d like to confirm that it’s no longer being used for anything related to that.

Could you please review my domain and consider removing it from the blacklist? Let me know if you need any more details from me.

Thanks a lot for your help! Let me know immediately if you need any more information or have concerns.

I’m not fluent in English and used AI to assist with this issue. Please excuse any errors in grammar or wording.

<hr>

<div align="center">
  <sub><a href="https://doh.tiar.app" target="_blank">Tiarap ⚡️</a></sub>
</div>

**Comment by SALTWOOD** [comment]
It's my only personal domain, and it was originally used for hosting open-source programs. The BT tracker server that ran on the domain was used for researching principles, and after completing my research, I **unfortunately** didn’t turn it off in time.
So I turned it off and `rm -rf bt-tracker` **just now**.

**Comment by pengelana** [comment]
`saltwood.top` is not blocked and resolved normally.

~~~
$ drill @174.138.21.128 saltwood.top
;; ->>HEADER<<- opcode: QUERY, rcode: NOERROR, id: 42745
;; flags: qr rd ra ; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 0
;; QUESTION SECTION:
;; saltwood.top.        IN      A

;; ANSWER SECTION:
saltwood.top.   39      IN      A       116.16.44.10

;; AUTHORITY SECTION:

;; ADDITIONAL SECTION:

;; Query time: 6 msec
;; SERVER: 174.138.21.128
;; WHEN: Mon Dec  2 00:19:24 2024
;; MSG SIZE  rcvd: 46
~~~

**Comment by SALTWOOD** [comment]
OK. Thank you.

**Comment by pengelana** [comment]
No, `saltwood.top` is not in the blocklist.

Your server is not responding to the request.

~~~
$ curl -vSLI http://saltwood.top
* Rebuilt URL to: http://saltwood.top/
*   Trying 116.16.44.10...
* TCP_NODELAY set
* Connection failed
* connect to 116.16.44.10 port 80 failed: Operation timed out
* Failed to connect to saltwood.top port 80: Operation timed out
* Closing connection 0

$ curl -vSLI https://saltwood.top
* Rebuilt URL to: https://saltwood.top/
*   Trying 116.16.44.10...
* TCP_NODELAY set
* Connection failed
* connect to 116.16.44.10 port 443 failed: Operation timed out
* Failed to connect to saltwood.top port 443: Operation timed out
* Closing connection 0
curl: (7) Failed to connect to saltwood.top port 443: Operation timed out
~~~

**Comment by SALTWOOD** [comment]
Okay. I recently searched for my domain name on GitHub and found it mentioned in the code of several repositories. Since my English skills aren't very strong, I mistakenly posted an issue in repositories containing words like "blocklist."

After carefully reviewing your repository, I noticed that the blocked word list is synchronized from other repositories via GitHub Actions, and the upstream repository has already resolved the issue. Therefore, my problem isn't related to your repository at all.

I apologize for the inconvenience and thank you for your understanding!
